packages/select-menu/src/index.tsx

```diff
@@ -115,6 +122,10 @@ class SelectMenuC<T extends OptionBase> extends React.Component<
   }
   isOptionSelected = (option: string) => {
     if (!this.props.isMultiSelect) {
+      let el = document.getElementById(this.props.value as any)
```

Comment:
Something like this would be better. Your solution will break when you have 2 or more selects with the same options(or at least one).
```
this.menuListRef.current.children.find(item => item.value === this.props.value)
```